components/page-header.tsx

```diff
@@ -1,12 +1,30 @@
+"use client";
 import { ExternalLink } from "lucide-react";
+import { useState, useEffect } from 'react';
+
 
 export function PageHeader() {
+
+  const [characterName, setCharacterName] = useState('');
+
+  useEffect(() => {
+    document.title = characterName ? `${characterName} - Aeternum Tracker` : "Track and Customize Your New World Progress - Aeternum Tracker";
+  }, [characterName]);
+
   return (
     <header className="fixed left-0 top-0 z-50 flex w-full justify-center border-b  bg-gradient-to-b py-2 backdrop-blur-2xl border-neutral-800 bg-zinc-800/30 from-inherit">
       <div className="container mx-auto flex items-center gap-2">
         <p className="text-lg md:text-2xl font-extrabold tracking-tight grow whitespace-nowrap">
           Aeternum Tracker
         </p>
+
+        <input
```

Comment:
There is a `Input` component which is already styled. See https://github.com/lmachens/aeternum-tracker/blob/main/components/ui/input.tsx

Reply:
Thanks for pointing that out.